### PR TITLE
Release 0.19.1

### PR DESCRIPTION
**Pull request by marinagmoreira**
https://babelfish.arc.nasa.gov/confluence/display/FFFSW/IRG-FFTEST302+-+2024-07-02+-+0.19.1+-+Astrobee+Release+Testing

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Releases
 
+##Release 0.19.1
```
Comment (bcoltin) [inline RELEASE.md:3]: Missing space (also noticing we have not been consistent with # vs ## below...)

File: astrobee/config/localization.config [hunk 2]
```diff
@@ -29,11 +29,11 @@ brisk_num_ransac_iterations = 100
 brisk_early_break_landmarks = 100
 brisk_histogram_equalization = 1
 brisk_check_essential_matrix = false
-brisk_essential_ransac_iterations = 2000
+brisk_essential_ransac_iterations = 0
```
Comment (bcoltin) [inline astrobee/config/localization.config:32]: is this really right? (just triple checking...)

**Review by bcoltin — APPROVED** [review]

File: astrobee/config/localization.config (hunk 2 above)
Comment (marinagmoreira) [reply astrobee/config/localization.config:32]: @rsoussan ?

Comment (rsoussan) [reply astrobee/config/localization.config:32]: Oh yeah that's fine, is ignored either way since it's set to false in the line above

**Pull request merged**